spec/features/samples/sample_accession_errors_spec.rb

```diff
@@ -18,10 +18,6 @@
       it 'shows informative errors when required fields are missing for accessioning' do
```

Comment:
Does this test need removal / redoing, its not checking for 'informative errors' any more.